### PR TITLE
Add auth screens

### PR DESCRIPTION
**Pull request by zeraye**
Add `signIn` and `signOut` screens with error handling. You can see in the video below:

https://github.com/flatly-pw/mobile-app/assets/53649257/3c7ddff1-5713-48ad-af26-d2f2547ffb4f

### CODE REVIEW

File: features/AuthScreen/SignUpScreen/SignUpScreen.tsx [hunk 1]
```diff
@@ -1,41 +1,157 @@
...
+  const nameHandler = (newName: string) => {
+    setName(newName);
+  };
+
+  const nameErrorVisible = () => {
+    return !(name.length > 0);
+  };
+
+  const nameErrorText = () => {
+    return "Name should have at least one character";
```
Comment (kemeyr) [inline features/AuthScreen/SignUpScreen/SignUpScreen.tsx]: "Name cannot be empty" sounds better to me

File: features/AuthScreen/SignUpScreen/SignUpScreen.tsx [hunk 2]
```diff
@@ -1,41 +1,157 @@
...
+  const lastNameHandler = (newLastName: string) => {
+    setLastName(newLastName);
+  };
+
+  const lastNameErrorVisible = () => {
+    return !(lastName.length > 0);
+  };
+
+  const lastNameErrorText = () => {
+    return "Last Name should have at least one character";
```
Comment (kemeyr) [inline features/AuthScreen/SignUpScreen/SignUpScreen.tsx]: Same as above

Comment (zeraye) [reply features/AuthScreen/SignUpScreen/SignUpScreen.tsx]: Resolved in https://github.com/flatly-pw/mobile-app/pull/11/commits/3e958814e7e18b7b8f7d536cc81367d1ae558143

File: features/AuthScreen/SignUpScreen/SignUpScreen.tsx (hunk 1 above)
Comment (zeraye) [reply features/AuthScreen/SignUpScreen/SignUpScreen.tsx]: https://github.com/flatly-pw/mobile-app/pull/11/commits/3e958814e7e18b7b8f7d536cc81367d1ae558143

**Pull request merged**